src/gm.datepickerMultiSelect.js

```diff
@@ -48,7 +48,7 @@ SOFTWARE.
 
 					/* Fires when date is selected or when month is changed. */
 					scope.$watch(function () {
-						return ctrl.activeDate.getTime();
+						return ctrl.activeDate ? ctrl.activeDate.getTime() : false;
```

Comment:
Oh really? OK I think it's fine then.

If you would, I'd like to see it in a plunker before I merge.